nshlib/Kconfig

```diff
@@ -231,6 +231,13 @@ config NSH_ALIAS_MAX_AMOUNT
 
 endif # NSH_ALIAS
 
+config NSH_PIPELINE
+	bool "Enable pipeline support"
+	default n
```

Comment:
Why not keep default y ?

Reply:
> Why not keep default y ?

```Diff
-       default n
+       default !DEFAULT_SMALL
```

Reply:
@JianyuWang0623 in this case, it is better to use:
```
     default !DEFAULT_SMALL
```

OR

```     
     default y if !DEFAULT_SMALL
```

to make the code clear